src/components/feed/NodeDetails/NodeDetails.tsx

```diff
@@ -352,7 +352,7 @@ function getCommand(
     }
   }
 
-  let command = `$> docker run --rm -v $(pwd)/in:/incoming -v $(pwd)/out:/outgoing \\\n${dock_image} ${selfexec} `;
+  let command = `$> docker run -d -v $(pwd)/in:/incoming -v $(pwd)/out:/outgoing ${dock_image} \\\n$ container_id=$(docker ps -q --filter "ancestor=${dock_image}") \\\n$ docker exec $container_id ${selfexec}\\\n $docker stop $container_id \\\n`;
```

Comment:
@itsgurmannatsohal could you explain to me what the `apptainer exec` command is and how it differs from `docker run`?

Reply:
`apptainer exec` is a command used to manage containers and images within an organization. The `apptainer exec` command allows you to run a command within an existing container.

The `docker run` command is used to start a new container from an image. When we use `docker run`, a new container is created, and the command specified is executed within the container.

Reply:
Hmm, I don't see `apptainer exec` anywhere in your patch.

Apptainer is a completely separate and different container runtime from docker. See https://apptainer.org

The command-line usage of `apptainer exec` and `docker run` are slightly different. In some cases it's preferable to use `apptainer` instead of `docker` because:

- it is usually preinstalled on scientific compute cluster machines
- it does not require you to be the root user by default
- it has weaker filesystem isolation by binding (i.e. mounting as volumes) some host paths automatically, so that the container runs as your user and home directory (among other benefits)


Please compare and contrast `apptainer` from `docker` and then translate this `docker run` command to an `apptainer exec` command. A stretch goal to do here would be adding additional flags as necessary which specify the resource limits, such as using the `--nv` flag if the plugin instance's `gpu_limit` is greater than 0.

Reply:
Sorry I misunderstood the code a bit. I've made the changes, please review.